Libs/MRML/Core/vtkMRMLSliceCompositeNode.cxx

```diff
@@ -287,6 +287,9 @@ void vtkMRMLSliceCompositeNode::CopyContent(vtkMRMLNode* anode, bool deepCopy/*=
   vtkMRMLCopyIntMacro(FiducialVisibility);
   vtkMRMLCopyIntMacro(FiducialLabelVisibility);
   vtkMRMLCopyIntMacro(SliceIntersectionVisibility);
+  // To avoid breaking current implementation, copy of the "LayoutName" attribute
+  // will be enabled after revisiting the view initilization pipeline.
+  //vtkMRMLCopyStringMacro(LayoutName);
```

Comment:
@lassoan  I will submit the use of references for background/foreground/layer in `vtkMRMLSliceCompositeNode` in a different PR, in the meantime I would like to move foward with the integration of this one.

Is the comment I integrated above  sensible ?